Warning when building rack

```
In file included from src/SurgePatchPlayer.cpp:1:
In file included from src/SurgePatchPlayer.hpp:3:
In file included from src/SurgeModuleCommon.hpp:12:
In file included from surge/src/common/SurgeStorage.h:18:
surge/src/common/Parameter.h:240:9: warning: private field 'axis' is not used [-Wunused-private-field]
   Axis axis;
```

Why does rack warn on that? Is it actually not used?

--- a/azure-pipelines.yml
+++ b/azure-pipelines.yml
@@ -187,7 +187,8 @@ jobs:
       sudo apt-get install -y clang
 
       cmake -Bbuild -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++
-      cmake --build build --config Release 
+      # Build the LV2 since that's what the freebsd folks use, and I build the VSt3 on mac clang all the time
+      cmake --build build --config Release --target Surge-LV2-Packaged
 
     condition: variables.isLinuxClang
     displayName: linux - clang

--- a/src/common/Parameter.cpp
+++ b/src/common/Parameter.cpp
@@ -27,11 +27,11 @@
 #include <UserDefaults.h>
 #include "DebugHelpers.h"
 
-Parameter::Parameter() : posx( PositionHolder::Axis::X ),
-                         posy( PositionHolder::Axis::Y ),
-                         posy_offset( PositionHolder::Axis::YOFF )
+Parameter::Parameter()
 {
    val.i = 0;
+   posx = 0;
+   posy = 0;
    posy_offset = 0;
    storage = nullptr;
 }

--- a/src/common/Parameter.h
+++ b/src/common/Parameter.h
@@ -216,31 +216,6 @@ struct ParameterIDCounter
    }
 };
 
-/*
-** Similarly the "position" is part of the parameter and that's a nasty mistake - there should
-** be an indirection. Ideally we would remove posx and posy as members altogether but to allow
-** an incremental approach make a little class which casts to an int and can redirect (at some
-** future point) for position. At this current iteration it is simply a holder for an int.
-*/
-class PositionHolder
-{
-public:
-   typedef enum {
-      X,
-      Y,
-      YOFF
-   } Axis;
-   
-   PositionHolder(Axis a) : val(-1), axis(a)  { }
-
-   operator int() const { return val; }
-   PositionHolder& operator=(const int v) { val = v; return *this; }
-   
-private:
-   Axis axis;
-   int val;
-};
-
 // used to make the infowindow
 struct ModulationDisplayInfoWindowStrings {
    std::string val;
@@ -340,7 +315,7 @@ public:
    int valtype = 0;
    int scene; // 0 = patch, 1 = scene A, 2 = scene B
    int ctrltype;
-   PositionHolder posx, posy, posy_offset;
+   int posx, posy, posy_offset;
    ControlGroup ctrlgroup = cg_GLOBAL;
    int ctrlgroup_entry = 0;
    int ctrlstyle = cs_off;

--- a/src/common/gui/CScalableBitmap.cpp
+++ b/src/common/gui/CScalableBitmap.cpp
@@ -203,7 +203,6 @@ CScalableBitmap::CScalableBitmap(CResourceDescription desc, VSTGUI::CFrame* f)
 CScalableBitmap::CScalableBitmap(std::string ifname, VSTGUI::CFrame* f)
    : CBitmap(CResourceDescription(0)), svgImage(nullptr), frame(f)
 {
-   std::cout << "CSB for " << ifname << std::endl;
 #ifdef INSTRUMENT_UI   
     Surge::Debug::record( "CScalableBitmap::CScalableBitmap file" );
 #endif
@@ -216,7 +215,6 @@ CScalableBitmap::CScalableBitmap(std::string ifname, VSTGUI::CFrame* f)
     std::string extension = "svg";
     if( fname.length() > 3 )
        extension = fname.substr( fname.length() - 3 );
-    std::cout << "extension is " << extension << std::endl;
 
     //pngImage = nullptr;
     if( _stricmp( extension.c_str(), "svg" ) == 0 )

